ARCHITECTURE.md

```diff
@@ -433,6 +434,22 @@ experience for offline users;
 The only drawback of doing this manually is importing the fonts in the `assets/` folder (size shouldn't be relevant to
 impact even low-end devices), and specifying them in the `pubspec.yaml`.
 
+## Importing Fonts
```

Comment:
This docs could also mention the image assets, since the approach is the same right?

Reply:
I don't think image is useful, because it's pretty much straightforward, contrary to fonts, which has this specific naming that I had to highlight, because the only way to know about this is to actually go into `fonts_test.dart` and check it out a utility function